config.json

```diff
@@ -2379,6 +2379,25 @@
           "pattern-matching"
         ],
         "difficulty": 9
+      },
+      {
+        "slug": "zebra-puzzle",
+        "name": "Zebra Puzzle",
+        "uuid": "0a561b08-6e61-4415-a3f1-e7d411a4a0e2",
+        "prerequisites": [
+          "pattern-matching",
+          "case",
+          "cond",
+          "if",
+          "enum",
+          "maps",
+          "atoms"
+        ],
+        "practices": [
+          "enum",
+          "cond"
```

Comment:
We need to make sure that adding these doesn't put us over the 8 exercise limit for the concept exercise @angelikatyborska

Reply:
I completely forgot about this 😱 was the limit 8 or 10? If the limit is 8, then we need to rethink 7 concepts. I think we can do that on a separate branch. I wouldn't want to block @jiegillet PR because of our previous carelessness 😅 

```
~*~*~*~ enum ~*~*~*~

Practiced by:
10
[
  'rna-transcription',
  'resistor-color',
  'raindrops',
  'sum-of-multiples',
  'anagram',
  'hamming',
  'transpose',
  'connect',
  'minesweeper',
  'zebra-puzzle'
]

~*~*~*~ integers ~*~*~*~

Practiced by:
9
[
  'collatz-conjecture',
  'nth-prime',
  'leap',
  'all-your-base',
  'change',
  'luhn',
  'armstrong-numbers',
  'largest-series-product',
  'clock'
]

~*~*~*~ maps ~*~*~*~

Practiced by:
9
[
  'nucleotide-count',
  'scrabble-score',
  'grade-school',
  'matrix',
  'tournament',
  'binary-search-tree',
  'kindergarten-garden',
  'custom-set',
  'saddle-points'
]

~*~*~*~ pattern-matching ~*~*~*~

Practiced by:
11
[
  'bowling',
  'protein-translation',
  'triangle',
  'tournament',
  'meetup',
  'perfect-numbers',
  'queen-attack',
  'poker',
  'alphametics',
  'wordy',
  'pov'
]

~*~*~*~ ranges ~*~*~*~

Practiced by:
10
[
  'rotational-cipher',
  'pangram',
  'sum-of-multiples',
  'nth-prime',
  'isbn-verifier',
  'grains',
  'rail-fence-cipher',
  'diffie-hellman',
  'palindrome-products',
  'difference-of-squares'
]

~*~*~*~ recursion ~*~*~*~

Practiced by:
11
[
  'roman-numerals',
  'strain',
  'accumulate',
  'matching-brackets',
  'binary-search',
  'binary-search-tree',
  'prime-factors',
  'pascals-triangle',
  'spiral-matrix',
  'sieve',
  'pov'
]

~*~*~*~ regular-expressions ~*~*~*~

Practiced by:
9
[
  'word-count',
  'markdown',
  'forth',
  'acronym',
  'run-length-encoding',
  'isogram',
  'phone-number',
  'isbn-verifier',
  'grep'
]
```

Reply:
I think the limit is 10: https://github.com/exercism/docs/blob/ede9d4268d6de02255ccf91aac006197135f3de1/building/configlet/lint.md#L127